src/test_application/activescenes/scenarios_physicstest.cpp

```diff
@@ -108,9 +121,8 @@ struct PhysicsTestScene
     osp::active::ActiveEnt          m_hierRoot;
```

Comment:
Code scanning

Variable 'testapp::physicstest::PhysicsTestScene::m_hierRoot' is uninitialized. Always initialize a member variable (type.6).

Warning
Variable 'testapp::physicstest::PhysicsTestScene::m_hierRoot' is uninitialized. Always initialize a member variable (type.6).